utils/test/cli_installer_tests.py

```diff
@@ -409,8 +409,7 @@ def test_get_hosts_to_run_on1(self, load_facts_mock, run_playbook_mock):
         result = self.runner.invoke(cli.cli, self.cli_args)
 
         if result.exception is None or result.exit_code != 1:
-            print("Exit code: %s" % result.exit_code)
-            self.fail("Unexpected CLI return")
+            self.fail("Unexpected CLI return. Exit code: %s" % result.exit_code)
```

Comment:
Yes, probably. I didn't indent to change that here though. There are many other occurrences we can tackle in a separate effort.